### PR TITLE
EdgarEdgar

### PR DESCRIPTION
**Pull request by Cubbal25**

### CODE REVIEW

File: source/index.html.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ code_clipboard: true
 
 # Introduction
 
-Welcome to the Kittn API! You can use our API to access Kittn API endpoints, which can get information on various cats, kittens, and breeds in our database.
+Welcome to the Edgar API! You can use our API to access Edgar API endpoints, which can get information on various E D G A R in our database.
```
Comment (zacr) [inline source/index.html.md]: This is an example of a code review. Everything you wrote looks good here. Change one more place in the code as well.

**Review by zacr — CHANGES_REQUESTED** [review]

File: source/index.html.md (hunk 1 above)
Comment (Cubbal25) [reply source/index.html.md]: Added some crazy!

**Comment by zacr** [comment]
Merged!

File: source/index.html.md (hunk 1 above)
Comment (Cubbal25) [reply source/index.html.md]: http://54.146.92.27:4567/?shell#introduction
It's Up!

**Pull request merged**